renderer/viewer/lib/workerProxy.ts

```
@@ -41,11 +51,25 @@ export const useWorkerProxy = <T extends { __workerProxy: Record<string, (...arg
       }
       return (...args: any[]) => {
+        const msgId = messageId++
         const transfer = autoTransfer ? args.filter(arg => arg instanceof ArrayBuffer || arg instanceof MessagePort || arg instanceof ImageBitmap || arg instanceof OffscreenCanvas || arg instanceof ImageData) : []
         worker.postMessage({
           type: prop,
+          msgId,
           args,
         }, transfer as any[])
+        return {
+          // eslint-disable-next-line unicorn/no-thenable
+          then (onfulfilled: (value: any) => void) {
```

Biome (1.9.4)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)